antennatracker/source/docs/configuration.rst

```diff
@@ -13,7 +13,7 @@ antenna tracker. Below are the details of this set-up.
 Setting the AHRS orientation
 ============================
 
-If the flight controller board could not be mounted so that it's forward
+If the flight controller board could not be mounted so that its forward
 direction aligned with the antenna (as mentioned on the :ref:`Assembling a frame page <frame-assembly>`) the AHRS_ORIENTATION parameter should
```

Comment:
could you change "direction aligned" to "direction is aligned"? might as well fix that also

Reply:
reveiwed again, going to merge now and quick fix myself..thanks for the contribution

Reply:
You were too fast for me! Let me know if there's anything else you want me to fix.